js/src/carousel.js

```diff
@@ -83,19 +83,19 @@ const KEY_TO_DIRECTION = {
 const Default = {
   interval: 5000,
   keyboard: true,
-  slide: false,
   pause: 'hover',
-  wrap: true,
-  touch: true
+  ride: false,
+  touch: true,
+  wrap: true
 }
 
 const DefaultType = {
   interval: '(number|boolean)',
   keyboard: 'boolean',
-  slide: '(boolean|string)',
+  ride: '(boolean|string)',
```

Comment:
Should be one line below ? -> to Bs

Reply:
Yes it was my comment in the original Bootstrap PR. The change has been made only in the previous `const` apparently. I'll create a PR for that.